### PR TITLE
Easier fix for login_with_tokens()

### PR DESCRIPTION
**Pull request by AumJavalgikar**

### CODE REVIEW

**Comment by doluk** [comment]
@Kuchenmampfer @majordoobie we have currently no checks if the passed keys are valid and suitable for the ip. Should we add that with the next release?

File: coc/client.py [hunk 1]
```diff
@@ -329,10 +329,10 @@ async def login_with_tokens(self, *tokens: str) -> None:
         tokens: list[str]
             Tokens as found from https://developer.clashofclans.com under "My account" -> <your key> -> "token".
         """
+        self.correct_key_count = len(tokens)
```
Comment (doluk) [inline coc/client.py:332]: please update `self.key_count` too. Just to keep everything consistent.

**Review by doluk — CHANGES_REQUESTED** [review]
Looks good! Just one minor change. Can you please add it to the login_with_keys too? It is depreciated, but still there and affected by the same bug.

File: coc/client.py (hunk 1 above)
Comment (AumJavalgikar) [reply coc/client.py:332]: Done!

**Pull request merged**